src/tools/updater/main.cpp

```diff
@@ -93,7 +93,12 @@ class MyStudyFinder final : public Data::StudyFinder
                     study->parameters.readonly = true;
 
                 logs.info() << "Saving...";
-                study->saveToFolder(folder);
+
+                uint tempVersionVar = (uint)study->header.version;
```

Comment:
**tempVersionVar** : we should be more clear. This variable is the current study version, isn't it ?
So let's call it **currentStudyVersion**